src/pouchdb/sendDbByEmail.ts

```diff
@@ -0,0 +1,146 @@
+import { Alert, PermissionsAndroid, Platform } from 'react-native'
+import Mailer from 'react-native-mail'
+import RNFS from 'react-native-fs'
+import RNFetchBlob from 'rn-fetch-blob'
+import DeviceInfo from 'react-native-device-info'
+
+import type CozyClient from 'cozy-client'
+import Minilog from 'cozy-minilog'
+
+import { fetchSupportMail } from '/app/domain/logger/supportEmail'
+import {
+  hideSplashScreen,
+  showSplashScreen,
+  splashScreens
+} from '/app/theme/SplashScreenService'
+import { getInstanceAndFqdnFromClient } from '/libs/client'
+import { getErrorMessage } from '/libs/functions/getErrorMessage'
+
+const log = Minilog('🗒️ DB Mailer')
+
+export const sendDbByEmail = async (client?: CozyClient): Promise<void> => {
+  log.info('Send DB by email')
+
+  if (!client) {
+    log.info('SendDbByEmail called with no client, return')
+    return
+  }
+
+  try {
+    const permission = PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE
+    await PermissionsAndroid.request(permission)
+
+    const supportEmail = await fetchSupportMail(client)
+
+    const { fqdn } = getInstanceAndFqdnFromClient(client)
+
+    const instance = client.getStackClient().uri ?? 'not logged app'
+
+    const subject = `DB files for ${instance}`
+
+    const files = await RNFS.readDir(RNFS.DocumentDirectoryPath)
+
+    const dbFiles = files.filter(f => f.name.startsWith(`${fqdn}_`))
+
+    const externalFiles = []
+    for (const dbFile of dbFiles) {
+      const dirs = RNFetchBlob.fs.dirs
+
+      const internalPath = dbFile.path
+
+      if (Platform.OS === 'android') {
+        const date = Number(new Date())
+        const externalPath = `${dirs.DCIMDir}/DbFile_${dbFile.name}${date}.sqlite`
+
+        await RNFS.copyFile(internalPath, externalPath)
```

Comment:
It does not work with the external path ?

Reply:
do you mean "internal" path?

What I understand is that the Mail app cannot access the FlagshipApp's internal path to load attachments. So I have to copy them in the external path

Note that this behavior is different in the Android emulator and I didn't find a way to make this work in that env (but this is not problematic for now)